commons/typed-values/src/test/java/org/operaton/bpm/engine/test/variables/FileValueTypeImplTest.java

```diff
@@ -144,35 +144,25 @@ void createValueWithProperties() {
   @Test
   void createValueWithNullProperties() {
     // given
-    InputStream file = this.getClass().getClassLoader().getResourceAsStream("org/operaton/bpm/engine/test/variables/simpleFile.txt");
+    final InputStream file = this.getClass().getClassLoader().getResourceAsStream("org/operaton/bpm/engine/test/variables/simpleFile.txt");
     Map<String, Object> properties = new HashMap<>();
     properties.put("filename", "someFileName");
     properties.put("mimeType", null);
     properties.put("encoding", "someEncoding");
 
-    // when
-    try {
-      type.createValue(file, properties);
-      fail("expected exception");
-    } catch (IllegalArgumentException e) {
-      // then
-      assertThat(e.getMessage()).contains("The provided mime type is null. Set a non-null value info property with key 'filename'");
-    }
+    assertThatThrownBy(() -> type.createValue(file, properties), "expected exception")
```

Comment:
The message "expected exception" does not add value anymore. Please remove all occurrences.
```suggestion
    assertThatThrownBy(() -> type.createValue(file, properties))
```